viz_scripts/scaffolding.py

```diff
@@ -125,12 +157,32 @@ async def load_viz_notebook_data(year, month, program, study_type, dynamic_label
     labeled_ct = filter_labeled_trips(participant_ct_df)
     expanded_ct = expand_userinputs(labeled_ct)
     expanded_ct = data_quality_check(expanded_ct)
+    expanded_ct = await map_trip_data(expanded_ct, study_type, dynamic_labels, dic_re, dic_pur)
+
+    # Document data quality
+    file_suffix = get_file_suffix(year, month, program)
+    quality_text = get_quality_text(participant_ct_df, expanded_ct, None, include_test_users)
+
+    debug_df = pd.DataFrame.from_dict({
+            "year": year,
+            "month": month,
+            "Registered_participants": len(get_participant_uuids(program, include_test_users)),
+            "Participants_with_at_least_one_trip": unique_users(participant_ct_df),
+            "Participant_with_at_least_one_labeled_trip": unique_users(labeled_ct),
+            "Trips_with_at_least_one_label": len(labeled_ct),
+            "Trips_with_mode_confirm_label": trip_label_count("Mode_confirm", expanded_ct),
+            "Trips_with_trip_purpose_label": trip_label_count("Trip_purpose", expanded_ct)
+            },
+        orient='index', columns=["value"])
+
+    return expanded_ct, file_suffix, quality_text, debug_df
 
+async def map_trip_data(expanded_trip_df, study_type, dynamic_labels, dic_re, dic_pur):
```

Comment:
I understand the reason for refactoring and pulling this out, but wonder why you chose to rename the parameter. It has lead to a lot of changed lines that are essentially just the variable rename. Not asking for a change, just an explanation.

Reply:
We are calling `map_trip_data(expanded_trip_df, ...)` from two functions -
`load_viz_notebook_inferred_data(expanded_it, ...)` & `load_viz_notebook_data(expanded_ct, ...)`. 
Therefore, I wanted to use a variable name which would suit both the parameters.